docs/source/kv_cache/cpu_ram.rst

```diff
@@ -1,4 +1,274 @@
 CPU RAM
 =======
 
-Coming soon... 
\ No newline at end of file
+Overview
+--------
+
+CPU RAM and Local Storage are the two ways of offloading KV cache onto non-GPU
+memory of the same machine that is running inference.
+
+Two ways to configure LMCache CPU Offloading:
+---------------------------------------------
+
+**1. Environment Variables:**
+
+``LMCACHE_USE_EXPERIMENTAL`` MUST be set by environment variable directly.
+
+.. code-block:: bash
+
+    # Specify LMCache V1
+    export LMCACHE_USE_EXPERIMENTAL=True
+    # 256 Tokens per KV Chunk
+    export LMCACHE_CHUNK_SIZE=256
+    # Enable CPU memory backend
+    export LMCACHE_LOCAL_CPU=True # default
+    # 5GB of Pinned CPU memory
+    export LMCACHE_MAX_LOCAL_CPU_SIZE=5.0 # default
+
+**2. Configuration File**:
+
+Passed in through ``LMCACHE_CONFIG_FILE=your-lmcache-config.yaml``
+
+``LMCACHE_USE_EXPERIMENTAL`` MUST be set by environment variable directly.
+
+Example ``config.yaml``:
+
+.. code-block:: yaml
+
+    # 256 Tokens per KV Chunk
+    chunk_size: 256
+    # Enable CPU memory backend
+    local_cpu: true # default
+    # 5GB of Pinned CPU memory
+    max_local_cpu_size: 5.0 # default
+
+CPU RAM Explanation:
```

Comment:
Actually, can you fix the title underlines so that Sphinx doesn't throw warnings? I know the code still builds but it's hard to find real errors later if there are 100s of warnings